about.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<meta name="description" content="Labyrinth | About">
+	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
+
+	<!-- UI Sheets -->
+	<link rel="stylesheet" type="text/css" href="css/materialize.min.css">
+	<link rel="stylesheet" type="text/css" href="css/fonts.css">
+	<link rel="stylesheet" type="text/css" href="css/custom.css">
+	<link rel="stylesheet" type="text/css" href="css/footer.css">
+	<link rel="stylesheet" type="text/css" href="css/tiles.css">
+	<script type="text/javascript" src="js/jquery.translate.js"></script>
+	<script type="text/javascript" src="js/sweetalert2.min.js"></script>
+	<script type="text/javascript" src="js/translate.js"></script>
+	<!-- Engine Scripts -->
+	<script type="text/javascript" src="js/jquery.min.js"></script>
+	<script type="text/javascript" src="js/materialize.min.js"></script>
+	<script type="text/javascript" src="js/sound.js"></script>
+	<script type="text/javascript" src="js/tiles.js"></script>
+	<script type="text/javascript" src="js/levels.js"></script>
+	<script type="text/javascript" src="js/inventory.js"></script>
+	<script type="text/javascript" src="js/player.js"></script>
```

Comment:
are these scripts necessary?

Reply:
I am asking if you need them because having scripts in other files means that these files need to be checked when the script is changed in case we have (and we have no) specification